<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClockWidget</class>
 <widget class="QWidget" name="ClockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>660</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupB">
     <property name="title">
      <string>Black</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Knights::Clock" name="clockB" native="true"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeB">
        <property name="font">
         <font>
          <pointsize>32</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="displayFormat">
         <string>H:mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressB">
        <property name="value">
         <number>24</number>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupW">
     <property name="title">
      <string>White</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="Knights::Clock" name="clockW" native="true"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeW">
        <property name="font">
         <font>
          <pointsize>32</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximumTime">
         <time>
          <hour>22</hour>
          <minute>59</minute>
          <second>59</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>H:mm:ss</string>
        </property>
        <property name="timeSpec">
         <enum>Qt::LocalTime</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressW">
        <property name="value">
         <number>24</number>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Knights::Clock</class>
   <extends>QWidget</extends>
   <header location="global">clock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
